consensus/pacemaker/pacemaker.go

```diff
@@ -36,30 +42,25 @@ type Pacemaker interface {
 	InterruptRound(reason string)
 }
 
-var (
-	_ modules.Module = &paceMaker{}
-	_ PacemakerDebug = &paceMaker{}
-)
-
-type paceMaker struct {
+type pacemaker struct {
 	bus            modules.Bus
 	pacemakerCfg   *configs.PacemakerConfig
 	stepCancelFunc context.CancelFunc
 
 	// Only used for development and debugging.
-	paceMakerDebug
+	pacemakerdebug
```

Comment:
Maybe `debug pacemakerDebug` will make it clear and explicit on what we're doing here.